.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Pull request
+about: Create a pull request for an issue in Altinn Studio
+---
+
+Thank you for contributing to Altinn:) We know this isn't the fun part, 
+but please make sure you follow our [contributing guidelines](../../CONTRIBUTING.md) and 
+put the same effort into the pull request as you did into the code and 
+it should soon find it's way to master.
+
+# {PR title}
+
+Summary of the changes (max 80 characters)
+
+## Description
+
+{Detail}
+## Fixes
+- #{issue number} (in this specific format)
```

Comment:
What do we mean with `(in this specific format)`? Not sure I get it.. :)

Reply:
I know what is meant , but `#{7319}` seems to be the obvious interpretation.

Reply:
I removed it.

Reply:
Anyway, just pasting the full github link to the issue gets shorted very nicely, so that's my recommendation.